include/zephyr/net/conn_mgr.h

```diff
@@ -13,11 +13,59 @@ extern "C" {
 
 #if defined(CONFIG_NET_CONNECTION_MANAGER)
 
+struct net_if;
+struct net_l2;
+
+/**
+ * @brief Resend either NET_L4_CONNECTED or NET_L4_DISCONNECTED depending on whether connectivity
+ * is currently available.
+ */
 void net_conn_mgr_resend_status(void);
 
+/**
+ * @brief Mark an iface to be ignored by conn_mgr.
+ *
+ * @param iface - iface to be ignored.
+ */
+void net_conn_mgr_ignore_iface(struct net_if *iface);
+
+/**
+ * @brief Stop ignoring an iface.
+ *
+ * @param iface - iface to no longer ignore.
+ */
+void net_conn_mgr_unignore_iface(struct net_if *iface);
+
+/**
+ * @brief Check whether the provided iface is currently ignored.
+ *
+ * @param iface - The iface to check.
+ * @return true if the iface is being ignored by conn_mgr.
+ * @return false otherwise.
```

Comment:
If you document individual return values, `@retval` should be used instead.

Reply:
Same with this